winter-website.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE HTML>
+
+<html lang="en-GB">
+
+<head>
+
+  <title>Winter's Website</title>
+  
+  <link rel="stylesheet" href="winter-website.css">
+
+</head>
+
+<body>
+	
+	<div class="navBar">
+				<a class="navLink" href="#">Home</a>
+				<a class="navLink" href="#">What I Do</a>
+				<a class="navLink" href="#">Things I've Made</a>
+				<a class="navLink" href="#">Contact</a>
+	</div>
+	
+	<div class="sectionAll">
+		<section class="aboutMeSection">
+			<h1>
+				Winter<br />
+				Sanchez
+			</h1>
+	
+			<h2>Hey, I'm Winter! I make websites and stuff!
+			Nonumy primis corrumpit et vix, mei at duis meliore gubergren, te vix feugiat vivendo. Ne pro diam putant vulputate, eu his equidem denique persecuti. Qui an veri dicit, cu summo harum vix. Omnesque copiosae eu eos, ea nisl propriae expetendis usu, antiopam disputationi pro et. Pri cu omnes aliquam constituto.</h2>
+		
+		</section>
+		
+		<section class="aboutMeImages">
+				<img src="https://i.imgur.com/XNiMDnX.jpg">
+				<img src="https://i.imgur.com/XNiMDnX.jpg">
+				<img src="https://i.imgur.com/XNiMDnX.jpg">
+				<img src="https://i.imgur.com/XNiMDnX.jpg">
+		</section>
+	</div>
+	
+	
+
+</body>
+
+<footer>
+	
+	<section class ="footerBar">
+		<h4>&copy;&nbsp;Winter Sanchez 2021</h4>
+	</section>
```

Comment:
not sure you even need section in footer

Reply:
this is correct!  a footer doesn't introduce a new section. have changed now  : ]